product_docs/docs/pgd/5/durability/commit-scope-rules.mdx

```diff
@@ -2,49 +2,49 @@
 
-The last part of this operation is the commit scope kind, which here is `GROUP COMMIT`, a synchronous two-phase commit which will be confirmed when any two nodes in the named group confirm they have flushed and made visible the transactions changes. 
+The last part of this operation is the commit scope kind, which in this example is `GROUP COMMIT`. `GROUP COMMIT` is a synchronous two-phase commit that's confirmed when any two nodes in the named group confirm they've flushed the transactions changes and made them visible.
 
 ## The commit scope group
 
-There are three kinds of commit scope group, `ANY`, `ALL` and `MAJORITY`. They are all followed by a parenthesized list of one or more groups, which combine to make a pool of nodes that this operation will apply to. This list can be preceded by `NOT` which inverts to pool to all other groups apart from those in the list. Witness nodes are not eligible to be included in this pool as they do not replicate data.
+There are three kinds of commit scope groups: `ANY`, `ALL`, and `MAJORITY`. They're all followed by a list of one or more groups in parentheses. This list of groups combines to make a pool of nodes this this operation applies to. This list can be preceded by `NOT`, which inverts to pool to all other groups apart from those in the list. Witness nodes aren't eligible to be included in this pool, as they don't replicate data.
 
-- `ANY n` &mdash; is followed by an integer value, "n". It translates to any "n" nodes in the listed groups nodes.
-- `ALL` &mdash; is followed by the groups and translates to all nodes in the listed groups nodes.
-- `MAJORITY` &mdash; is followed by the groups and translates to requiring a half, plus one, of the listed groups nodes to confirm to give a majority.
-- `ANY n NOT` &mdash; is followed by an integer value, "n". It translates to any "n" nodes that are not in the listed groups nodes.
-- `ALL NOT` &mdash; is followed by the groups and translates to all nodes that are not in the listed group's nodes.
-- `MAJORITY NOT` &mdash; is followed by the groups and translates to requiring a half, plus one, of the nodes that are not in the listed groups nodes to confirm to give a majority.
+- `ANY n` is followed by an integer value, `n`. It translates to any `n` nodes in the listed group's nodes.
+- `ALL` is followed by the groups and translates to all nodes in the listed groups nodes.
+- `MAJORITY` is followed by the groups and translates to requiring a half, plus one, of the listed group's nodes to confirm, to give a majority.
+- `ANY n NOT` is followed by an integer value, `n`. It translates to any `n` nodes that aren't in the listed group's nodes.
+- `ALL NOT` is followed by the groups and translates to all nodes that aren't in the listed group's nodes.
+- `MAJORITY NOT` is followed by the groups and translates to requiring a half, plus one, of the nodes that aren't in the listed group's nodes to confirm, to give a majority.
 
```

Comment:
The em dash treatment usually wouldn't have a lower-cased verb following. These read more like sentences than what I'd call our typical "definition list" that includes the em-dash. I think they read fine without the em-dash (easier to read).